misc/SWC_badge/badge_snippet.html

```diff
@@ -0,0 +1,4 @@
+<html>
+<body>
+<a href="http://software-carpentry.org"><img src="file:///D:/Documents/public_github_repos/communications/misc/SWC_badge/SWC_badge.png"></a>
```

Comment:
It is currently a local link. That is obviously not correct.

IMHO it should be a link to the logo hosted at a sensible URL on SWC's website, rather than deep in a github repo[1]. Hence my query over on the Issue as to how we get the logo put on the SWC webspace.


[1] Yes, I'm aware that the SWC website *is* a github repo, but that's not what I mean ;-)